tests/async/test_fetch_global.py

```diff
@@ -524,3 +524,23 @@ async def test_should_not_throw_when_fail_on_status_code_is_false(
     response = await request.fetch(server.EMPTY_PAGE)
     assert response.status == 404
     await request.dispose()
+
+
+async def test_should_follow_max_redirects(
+    playwright: Playwright, server: Server
+) -> None:
+    redirect_count = 0
+
+    def _handle_request(req: TestServerRequest) -> None:
+        nonlocal redirect_count
+        redirect_count += 1
+        req.setResponseCode(301),
```

Comment:
nit: why all the commas at the end?

Reply:
Good catch. Bad copy job. They came from a lambda tuple expression.